src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -121,6 +123,10 @@ const SpendingByCFDAContainer = ({ activeTab, scrollIntoView }) => {
     const [error, setError] = useState(false);
     const [sort, setSort] = useState('obligation');
     const [order, setOrder] = useState('desc');
+    const [modalData, setModalData] = useState(null);
+    const [showModal, setShowModal] = useState(false);
+    const [redirectModalURL, setRedirectModalURL] = useState('');
```

Comment:
This is duplicating application state. The source of truth for this is in redux. See `GlobalModalContainer`.